### DISCUSSION TITLE
Sayı kuralı sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#202] [Zeka Sorusu] [answered]
72496 -> 1315
62134 -> 97
85316 -> 167
yukarıdaki kurala göre soru işaretli yere yazılacak rakam nedir?
28439 -> ?

**Comment by ArvenAA** [comment, marked as answer]
Sayıların ilk üç rakamlarının toplamları başa, son iki rakamlarının toplamları ise sona yazılıyor. 2+8+4 = 14 ve 3+9=12 yani soru işareti yerine gelmesi gereken sayı 1412.
